sockeye/training.py

```diff
@@ -443,17 +447,24 @@ def _log_params(self):
         logger.info("Model parameters: %s", ", ".join(info))
         logger.info("Total # of parameters: %d", total_parameters)
 
-
     def _save_params(self, output_folder: str, checkpoint: int):
         """
-        Saves the parameters to disk.
+        Synchronizes parameters across devices, saves the parameters to disk, and updates self.params.
         """
-        arg_params, aux_params = self.module.get_params()  # sync aux params across devices
+        arg_params, aux_params = self.module.get_params()
         self.module.set_params(arg_params, aux_params)
         self.params = arg_params
         params_base_fname = C.PARAMS_NAME % checkpoint
         self.save_params_to_file(os.path.join(output_folder, params_base_fname))
 
+    def _load_params(self, output_folder: str, checkpoint: int):
+        """
+        Loads parameters from disk, sets self.params and module's parameters.
+        """
+        params_fname = os.path.join(output_folder, C.PARAMS_NAME % checkpoint)
+        self.load_params_from_file(params_fname)  # sets self.params
+        self.module.set_params(self.params, {})
```

Comment:
ah the second argument are the aux_params, right? I was a little confused by this for a sec. Could you make it a named arg?
```
self.module.set_params(self.params, aux_params={})
```

Reply:
good point, done